wfs-browser-input.py

```diff
@@ -0,0 +1,136 @@
+# chmod +x /PATH-TO/selenium/webdriver/common/linux/selenium-manager
+
+import sys
+import time
+from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+# ANSI Colors
+YELLOW = "\033[93m"
+BLUE = "\033[94m"
+GREEN = "\033[92m"
+RESET = "\033[0m"
+
+def colored_print(text, color):
+    return f"{color}{text}{RESET}"
+
+def get_user_input(default, prompt, color):
+    user_input = input(colored_print(prompt, color))
+    if not user_input:
+        return default
+    return user_input
+
+def main():
+    url = get_user_input('http://192.168.1.1', "Router's ip (default: 192.168.1.1): ", RESET)
+    print("\r")
+
+    expression = {b"error", b"incorrect", b"failure", b"try", b"again", b"invalid"}
+
+    u_name = get_user_input("username", "Username html element name (default: username): ", YELLOW)
+    u_el = get_user_input("i", "Is username element an id or a name? (i/n): ", YELLOW)
```

Comment:
To improve code readability, I think `u_name` should be renamed to `username_element_type`

Reply:
For me it's ok